ui/raidboss/data/05-shb/raid/e6s.js

```diff
@@ -7,6 +7,114 @@
   },
   timelineFile: 'e6s.txt',
   triggers: [
+    {
+      id: 'E6S Strike Spark',
+      regex: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4BD3', capture: false }),
+      regexDe: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4BD3', capture: false }),
+      regexFr: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4BD3', capture: false }),
+      regexJa: Regexes.startsUsing({ source: ['イフリート', 'ラクタパクシャ'], id: '4BD3', capture: false }),
+      delaySeconds: 11,
+      promise: function(data) {
+        let p = new Promise(async (res) => {
+          let combatantNames = null;
+
+          const ifritLocaleNames = {
+            en: 'Ifrit',
+            de: 'Ifrit',
+            fr: 'Ifrit',
+            ja: 'イフリート',
+          };
+
+          const raktapaksaLocaleNames = {
+            en: 'Raktapaksa',
+            de: 'Raktapaksa',
+            fr: 'Raktapaksa',
+            ja: 'ラクタパクシャ',
+          };
+
+          // select the 4 most recent Ifrit or Raktapaksa's depending on phase
+          if (data.phase === 'ifrit')
+            combatantNames = [ifritLocaleNames[data.lang]];
+          else
+            combatantNames = [raktapaksaLocaleNames[data.lang]];
+
+          let combatantData = await window.callOverlayHandler({
+            call: 'getCombatants',
+            names: combatantNames,
+          });
+
+          // if we could not retrieve combatant data, the
+          // trigger will not work, so just resume promise here.
+          if (!(combatantData !== null &&
+            combatantData.combatants &&
+            combatantData.combatants.length)) {
+            data.safeZone = null;
+            res();
+            return;
+          }
+
+          // we need to filter for the Ifrit with the highest ID
+          // since that one is always the safe spot.
+          let currentHighestCombatant =
+            combatantData.combatants.sort((a, b) => a.ID - b.ID).pop();
+
+          // all variation ranges for all the 9 ball positions for the kicking actors
+          // north      x: 96-104   y: 85-93
+          // northeast  x: 107-115  y: 85-93
+          // northwest  x: 85-93    y: 85-93
+          // east       x: 107-115  y: 96-104
+          // west       x: 85-93    y: 96-104
+          // south      x: 96-104   y: 107-115
+          // southeast  x: 107-115  y: 107-115
+          // southwest  x: 85-93    y: 107-115
+          let safeZoneObj1 = { en: '', de: '' };
```

Comment:
I think I only have one other cleanup comment here, and that's that you should either fill out every language here (just to avoid future bugs) or to change the logic such that you initialize these to null and do something different later.